include/openrave/kinbody.h

```diff
@@ -490,6 +517,7 @@ class OPENRAVE_API KinBody : public InterfaceBase
         float _fTransparency = 0; ///< value from 0-1 for the transparency of the rendered object, 0 is opaque
         bool _bVisible = true; ///< if true, geometry is visible as part of the 3d model (default is true)
         bool _bModifiable = true; ///< if true, object geometry can be dynamically modified (default is true)
+        boost::array<dReal, 6> _vCropContainerMarginsXYZXYZ = {0, 0, 0, 0, 0, 0};
```

Comment:
I think the 6 dim array has been hard to understand in the system and will be even harder if introduced into openrave this way. How about `Vector negCropContainerMargins, posCropContainerMargins`?